KK-DZ/src/app/modules/workers/components/delete-confirmation-dialog/delete-confirmation-dialog.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, ChangeDetectionStrategy, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { EmployeeEntity } from 'src/app/core/models/employee-entity';
+
+@Component({
+  selector: 'delete-confirmation-dialog',
```

Comment:
Pilnujmy prefixa który wyróznia nasze componenty od tych wbudowanych/material/zewnetrzne biblioteki
```suggestion
  selector: 'app-delete-confirmation-dialog',
```